.github/workflows/links-check.yml

```diff
@@ -0,0 +1,89 @@
+name: Internal Links Check
+
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+      - '*.md'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      # It will be possible to check only for the links in the changed files
+      # See: https://github.com/lycheeverse/lychee-action/issues/17
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.5.0
```

Comment:
For non-GitHub maintained actions, i.e. any that don't start with `uses: actions/`, please use the git SHA instead of a version number tag.